### PR TITLE
Bug 1955229: step-registry: add Calico workflow

### PR DESCRIPTION
**Pull request by vrutkovs**
Move e2e-aws-calico job in 4.8 to workflows
Ref: https://issues.redhat.com/browse/DPTP-2039

### CODE REVIEW

**Comment by vrutkovs** [comment]
/hold

```
level=error msg=Cluster operator machine-config Degraded is True with RequiredPoolsFailed: Unable to apply 4.8.0-0.ci-2021-05-06-092748: timed out waiting for the condition during syncRequiredMachineConfigPools: error pool master is not ready, retrying. Status: (pool degraded: true total: 3, ready 0, updated: 0, unavailable: 3) 
```

File: ci-operator/step-registry/network/calico/OWNERS
Comment (wking) [inline ci-operator/step-registry/network/calico/OWNERS:6]: Do we have a Calico maintainer we can add here?

File: ci-operator/step-registry/openshift/e2e/aws/calico/openshift-e2e-aws-calico-workflow.yaml
Comment (wking) [inline ci-operator/step-registry/openshift/e2e/aws/calico/openshift-e2e-aws-calico-workflow.yaml:13]: can we get bug links for these, and/or improved skipping in origin?

File: ci-operator/step-registry/network/calico/OWNERS
Comment (vrutkovs) [reply ci-operator/step-registry/network/calico/OWNERS:6]: We attempted to do that in https://github.com/openshift/release/pull/17380, but `openshift-bot` was not happy - owners must be in openshift org.

**Comment by vrutkovs** [comment]
/retest
Deprovision error

File: ci-operator/step-registry/openshift/e2e/aws/calico/openshift-e2e-aws-calico-workflow.yaml
Comment (vrutkovs) [reply ci-operator/step-registry/openshift/e2e/aws/calico/openshift-e2e-aws-calico-workflow.yaml]: Removed this skip, its now in master after https://bugzilla.redhat.com/show_bug.cgi?id=1957146 was fixed

**Comment by jluhrsen** [comment]
not that I have any authority here, but:
/lgtm

the [rehearsal is nice](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/18367/rehearse-18367-periodic-ci-openshift-release-master-ci-4.8-e2e-aws-calico/1394199883392487424), and especially since the existing job is [pointlessly failing](https://testgrid.k8s.io/redhat-openshift-ocp-release-4.8-informing#release-openshift-origin-installer-e2e-aws-calico-4.8).

**Review by petr-muller — APPROVED** [review]

**Pull request merged**
